net/xray-core/Makefile

```diff
@@ -1,18 +1,17 @@
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=xray-core
-PKG_VERSION:=1.4.5
+PKG_NAME:=Xray-core
```

Comment:
why did you do a rename? Never saw that people use capital chars in the pkg-name.

Reply:
It doesn't affect the real pkg-name:
- xray-core
- xray-example
- xray-geodata

In order to reuse default PKG_BUILD_DIR, I made this change.
[openblas](https://github.com/openwrt/packages/blob/master/libs/openblas/Makefile) also did the same thing.

Reply:
Even you're right and you found the exception, the pkg_name should match the naming of the folder. So, I'd suggest to drop this change.

Reply:
Updated, thanks.